### ISSUE TITLE
Content clicks don't register on mobile devices

### ISSUE THREAD

**Issue by steffchep** [bcherny/draggable#36]
Hi! 

Great working library, works really well. However, I noticed one thing on mobile browsers:
- if there are buttons inside the dragged content, and you've not specified a drag handle (i.e. making the element itself the handle), those buttons don't work at all - tapping them does nothing

Seen on Samsung Tab S6 Lite and Apple iPad Mini 5 (2019):
- Android 11 Samsung Internet Browser
- Android 11 Chrome
- Android 11 Firefox
- iPadOS 14.6 Safari
- iPadOS 14-6 Firefox

In summary, looks like touch events are handled a little differently from mouse events.

The obvious workaround is to use a drag handle whenever there is clickable content inside - I was just wondering if there is something else that can be done.

**Comment by steffchep** [comment]
Update: another workaround is using the filterTarget function in the options. Still it's odd that it would behave differently on Firefox, Edge or Chrome for Windows than it does on the listed mobile devices
